### DISCUSSION TITLE
Why it is recommended to prefer manned.org?

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [tldr-pages/tldr#7507] [Q&A] [answered]
Hello! I want to understand why it is recommended to prefer manned.org over [listed alternatives](https://manned.org/). Is it because count of man pages is bigger than for instance in [manpages.org/](http://manpages.org/)?

**Comment by navarroaxel** [comment, marked as answer]
We decide the usage of [manned.org](https://manned.org/) here https://github.com/tldr-pages/tldr/issues/5143.

**Comment by sbrl** [comment]
It's important to note here that this doesn't mean to say that man pages are preferred over other resources. Indeed, specific documentation about a command line program and also it's general homepage both come above a man page link.
